dist/stat_data.py

```diff
@@ -1199,6 +1199,7 @@ def __init__(self, name, desc, flags=''):
     BlockDisaggStat('disagg_block_hs_get', 'Disaggregated block manager get from the shared history store in SLS'),
     BlockDisaggStat('disagg_block_hs_put', 'Disaggregated block manager put to the shared history store in SLS'),
     BlockDisaggStat('disagg_block_put', 'Disaggregated block manager put '),
+    BlockDisaggStat('disagg_block_pages_discarded', 'Pages discarded by disaggregated block manager'),
```

Comment:
Please sort them alphabetically
```suggestion
    BlockDisaggStat('disagg_block_pages_discarded', 'Pages discarded by disaggregated block manager'),    BlockDisaggStat('disagg_block_put', 'Disaggregated block manager put '),

```

Reply:
Thanks for picking it up, changed in [e09ba12](https://github.com/wiredtiger/wiredtiger/pull/12032/commits/e09ba12391d50aad687479a49b22739a1d2b5960)